diff --git a/yt_dlp/cookies.py b/yt_dlp/cookies.py
index 4a69c576b..9949e4a91 100644
--- a/yt_dlp/cookies.py
+++ b/yt_dlp/cookies.py
@@ -154,7 +154,7 @@ def _extract_firefox_cookies(profile, container, logger):
             identities = json.load(containers).get('identities', [])
         container_id = next((context.get('userContextId') for context in identities if container in (
             context.get('name'),
-            try_call(lambda: re.fullmatch(r'userContext([^\.]+)\.label', context['l10nID']).group()),
+            try_call(lambda: re.fullmatch(r'user-context-([^\.]+)', context['l10nId']).group()),
         )), None)
         if not isinstance(container_id, int):
             raise ValueError(f'could not find firefox container "{container}" in containers.json')
